app/graphql/types/search_result/global_scope/target_type.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Types::SearchResult::GlobalScope::TargetType = GraphQL::UnionType.define do
```

Comment:
I'm not sure if `target` is the best choice for the name. When I read `target`, I think of what we call `category`. How about `SpecificEntry`?

It's also weird to have the field `global.entries.entry` to be a `GlobalSearchTarget` instead of something with "entry" in it ;)